config/default_records/pipelines/002_element_aviti_pipelines.yml

```diff
@@ -2,6 +2,7 @@
 Element Aviti:
   name: Element Aviti
   sti_type: SequencingPipeline
+  validator_class_name: ElementAvitiValidator
```

Comment:
N.B. If this is deployed after the submissions PR where this record is added, this will need to be manually updated.